core-bundle/src/Resources/contao/dca/tl_content.php

```diff
@@ -277,30 +277,30 @@
 			'inputType'               => 'imageSize',
 			'reference'               => &$GLOBALS['TL_LANG']['MSC'],
 			'eval'                    => array('rgxp'=>'natural', 'includeBlankOption'=>true, 'nospace'=>true, 'helpwizard'=>true, 'tl_class'=>'w50'),
-			'sql'                     => "varchar(255) NOT NULL default ''"
+			'sql'                     => "varchar(64) COLLATE ascii_bin NOT NULL default ''"
```

Comment:
I am not sure about this change. If you have an image size called `this_lorem_ipsum_foobar_image_size` then the resulting serialized array

```
a:3:{i:0;i:0;i:1;i:0;i:2;s:23:"_this_lorem_ipsum_foobar_image_size";}
```

would be 69 characters long. (nice)

Reply:
```suggestion
			'sql'                     => "varchar(128) COLLATE ascii_bin NOT NULL default ''"
```